adapter/src/ethereum/client.rs

```diff
@@ -723,19 +722,51 @@ mod test {
 
         let spender = *CREATOR;
 
+        // todo: Clean up
```

Comment:
There's one todo left. Is it for future reference?

Reply:
Yes, could be improved later on, it's only for the test.